src/help/technicalHistory.md

```diff
@@ -0,0 +1,48 @@
+#### January 18, 2017
```

Comment:
Dummy content to test that it is pulling files correctly. These will be modified manually be Kaitlin